config/settings.yml

```diff
@@ -52,11 +52,11 @@
     :agent_coordinator:
       :agent_ami_name: CentOS Atomic Host 7 x86_64 HVM EBS 1706_01
       :agent_ami_login_user: centos
-      # TODO: will replace with official docker image name
-      :agent_docker_name: docker.io/hsong/ssa_support
+      :agent_ami_registration_required: false
```

Comment:
We're not registering the ami, we're logging into a docker registry so maybe `:docker_login_required`?